src/App.js

```diff
@@ -0,0 +1,16 @@
+import { Route, Routes } from 'react-router-dom';
+import Home from './pages/Home';
+import Details from './pages/Details';
+
```

Comment:
- [ ] The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/158635985-2f52c333-74a9-4a90-ad7c-a8d17cfb6a45.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)